### ISSUE TITLE
No containers when viewing Pod with 1.6.0

### ISSUE THREAD

**Issue by benzman81** [rancher-sandbox/rancher-desktop#3162]
### Actual Behavior

If you click an on Pod, it does not show the belonging containers. Instead the view looks like this:

![RancherBug](https://user-images.githubusercontent.com/14224592/195880348-539006e0-4cd1-40db-9b39-309834afb905.png)

### Steps to Reproduce

- Go to Pods
- Click on a Pod
- no containers shown

### Result

Should show containers, but doesnt. See screenshot above.

### Expected Behavior

Should show containers like this:

![RancherBug2](https://user-images.githubusercontent.com/14224592/195880652-b412c493-3a28-4148-a929-773346331f4f.png)

### Additional Information

Worked with 1.5.0.

### Rancher Desktop Version

1.6.0

### Rancher Desktop K8s Version

v1.24.6

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 20H2

### What CPU architecture are you using?

x64

### Windows User Only

no relevant info.

**Comment by ericpromislow** [comment]
I can't see the container list in both versions 1.5.1 (on macOS) and 1.6.0 on Windows.
Where did the second screenshot come from? Like which version and platform?

**Comment by ericpromislow** [comment]
Thank you for confirming the regression.

**Comment by rak-phillip** [comment]
Taking another look at this one and I think the issue might stem from the version of Rancher Dashboard bundled with Rancher Desktop.

As described in the issue, this is what we would expect to see when selecting a pod.

![image](https://user-images.githubusercontent.com/835961/203647213-857c6ad8-d0e5-47b4-98f7-433c4421e28f.png)

I'm using this test deployment to replicate:

```
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    workload.user.cattle.io/workloadselector: apps.deployment-default-test-logs
  name: test-logs
  namespace: default
spec:
  progressDeadlineSeconds: 600
  replicas: 1
  revisionHistoryLimit: 10
  selector:
    matchLabels:
      workload.user.cattle.io/workloadselector: apps.deployment-default-test-logs
  strategy:
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
    type: RollingUpdate
  template:
    metadata:
      labels:
        workload.user.cattle.io/workloadselector: apps.deployment-default-test-logs
    spec:
      affinity: {}
      containers:
      - args:
        - 'i=0; while true; do echo "$i: $(date)"; i=$((i+1)); sleep .001; done'
        command:
        - /bin/sh
        - -c
        image: busybox
        imagePullPolicy: Always
        name: fast
        resources: {}
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
      - args:
        - 'i=0; while true; do echo "$i: $(date)"; i=$((i+1)); sleep 1; done'
        command:
        - /bin/sh
        - -c
        image: busybox
        imagePullPolicy: Always
        name: slower
        resources: {}
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
      - args:
        - 'i=0; while true; do echo "$i: $(date)"; i=$((i+1)); sleep 10; done'
        command:
        - /bin/sh
        - -c
        image: busybox
        imagePullPolicy: Always
        name: sloooow
        resources: {}
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
      dnsPolicy: ClusterFirst
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext: {}
      terminationGracePeriodSeconds: 30
```

**Comment by jandubois** [comment]
I believe this is fixed by #3454
